src/main/java/frc/robot/Robot.java

```diff
@@ -27,19 +33,30 @@
 
   Timer timer;
 
+  ButtonToggleRunner shifterToggleRunner;
+  ButtonToggleRunner intakeDeployerToggleRunner;
+  ButtonToggleRunner winchLockToggleRunner;
+
   @Override
   public void robotInit() {
     leftJoystick = new EnhancedJoystick(0);
     rightJoystick = new EnhancedJoystick(1);
     manipulator = new Gamepad(2);
 
-    driveBase = new DriveBase(0, 1, 0, 1, 2, 3, 0);
-    harvester = new Harvester(2, 1);
-    ballTransfer = new BallTransfer(3);
-    ballChucker9000 = new BallChucker9000(4, 5, 6, 4, 5, 6, 7, 2, 8);
-    climb = new Climb(7, 8, 9);
+
+    driveBase = new DriveBase(
+        new SpeedControllerGroup(new WPI_TalonSRX(10), new WPI_VictorSPX(11), new WPI_VictorSPX(12)),
+        new SpeedControllerGroup(new WPI_TalonSRX(20), new WPI_VictorSPX(21), new WPI_VictorSPX(22)), 4, 3, 0, 1, 2, 3);
+    harvester = new Harvester(new VictorSP(5), 2, 0);
+    ballTransfer = new BallTransfer(new VictorSP(3));
+    ballChucker9000 = new BallChucker9000(new VictorSP(0), new VictorSP(1), new WPI_VictorSPX(4), new WPI_VictorSPX(31), 5, 6, 7, 8, 9, 10);
```

Comment:
Can we check on whether we should use `new VictorSP()` or `new WPI_Victor_SP()`?